pom.xml

```diff
@@ -233,6 +233,10 @@
             <version>2.6</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
@yegor256 what do you mean by "optional"? Can it be excluded and we are sure that Takes will be working 100% fine?